src/themes/elife/index.ts

```diff
@@ -30,6 +31,7 @@ ready((): void => {
     dateFormatter.format(first(':--datePublished'))
     socialSharers.build(articleTitle, articleId)
     referenceFormatter.format(select(':--reference'))
+    externalLinks.format(select('a[href^=http]'))
```

Comment:
Is http correct or does it need to be https?

Reply:
This should select both `http` and `https`, so I think it should be good to go

Reply:
I have approved this on @alex-ketch comments